lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+  attr_accessor :cells
+
+
+  def initialize
+    self.reset!
+  end
+
+  def reset!
+    @cells = Array.new(9," ")
+  end
+
+  def position
+    x = gets.chomp.to_i
+    x-=1
+    @cells[x]
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} \n-----------\n #{@cells[3]} | #{@cells[4]} | #{@cells[5]} \n-----------\n #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+  #to display before the start of the game, making it user-friendly
+  def display_intro
+    puts " 1 | 2 | 3 \n-----------\n 4 | 5 | 6 \n-----------\n 7 | 8 | 9 "
+  end
+
+  def full?
+    !@cells.any?{|x| x == " " || x == "" }
```

Comment:
I understand. I did not think this through. I simply thought of the easiest way that I could implement it. :S